src/components/Block/__snapshots__/Block.test.tsx.snap

```diff
@@ -15,37 +19,64 @@ exports[`<Block /> with additional className matches its snapshot 1`] = `
 
 exports[`<Block /> with bottom spacing matches its snapshot 1`] = `
 <div
-  className="y-block y-block__bottomSpacing-xLarge"
-  style={Object {}}
+  class=
+      y-block
+      {
+        margin-bottom: 2rem;
+      }
 >
   <div
-    className="y-block--inner"
+    class=
+        y-block--inner
+        {
+          overflow-wrap: break-word;
+          word-wrap: break-word;
+        }
   >
     block content
   </div>
 </div>
 `;
 
 exports[`<Block /> with default options matches its snapshot 1`] = `
-<div
-  className="y-block"
-  style={Object {}}
+<[object Object]
```

Comment:
does the latest version of jest not fix this? or are we stuck waiting on enzyme's update?

Reply:
Latest version of jest AND replacing enzyme's `shallow` with react's shallow renderer https://reactjs.org/docs/shallow-renderer.html

I've made the change now, but I'm good either way.